docker/qr/http/index.html

```diff
@@ -4,6 +4,7 @@
 <head>
     <title>Quadradius</title>
     <meta charset="utf-8" />
+    <meta name="description" content="Making it possible to play Quadradius again" />
```

Comment:
I'd personally make it more descriptive. What is Quadradius? Maybe mention it's a game from 2007 + what it looks like.